core/sdk-core/src/main/java/software/amazon/awssdk/core/internal/async/FileAsyncRequestBody.java

```diff
@@ -271,34 +274,22 @@ private void closeFile() {
             }
```

Comment:
`closeFile` calls `signalOnError` if it fails, but we may have already called `onComplete` or `onError` which is a spec violation. Should we just log the error as a warning instead of signaling it?

Reply:
Ah, good catch! Yeah, we should just log the error